heron/stmgr/src/cpp/manager/stmgr-client.cpp

```diff
@@ -148,9 +148,6 @@ void StMgrClient::HandleHelloResponse(void*, proto::stmgr::StrMgrHelloResponse*
     Stop();
   }
   delete _response;
-  if (client_manager_->DidAnnounceBackPressure()) {
```

Comment:
follow up question:
how do you maintain the state eg. backpressure_starters_ after stmgr restarts/reconnects?